tests/ctst/world/Zenko.ts

```diff
@@ -497,20 +598,21 @@ export default class Zenko extends World {
             this.addCommandParameter({ policyName: Zenko.IAMUserPolicyName });
             this.addCommandParameter({ policyPath: '/' });
             if (process.env.POLICY_DOCUMENT) {
-                this.addCommandParameter({ policyDocument: JSON.parse(process.env.POLICY_DOCUMENT) });
+                this.addCommandParameter({ policyDocument: JSON.parse(process.env.POLICY_DOCUMENT) as object });
             } else {
                 this.addCommandParameter({ policyDocument: IAMUserPolicy });
             }
             const policy = await IAM.createPolicy(this.getCommandParameters());
             const account = await SuperAdmin.getAccount({
-                name: this.parameters.AccountName || Constants.ACCOUNT_NAME,
-            });
-            let policyArn = `arn:aws:iam::${account.id}:policy/IAMUserPolicy-${Zenko.IAMUserName}}`;
+                accountName: this.parameters.AccountName || Constants.ACCOUNT_NAME,
+            }) as Utils.Account['account'];
+            let policyArn = `arn:aws:iam::${account.id as string}:policy/IAMUserPolicy-${Zenko.IAMUserName}}`;
```

Comment:
if `account` is already casted, the `as` in line 609 should be needed, as it should be part of the `account` type already. If it complains because id can be null, you can do

```ts
            let policyArn = `arn:aws:iam::${account.id!}:policy/IAMUserPolicy-${Zenko.IAMUserName}}`;
```

(same for all related)

Reply:
Using the exclamation mark operator creates warnings from the linter, not that I agree with the rule as it would've made things easier to remove it in many places. 
```
  609:45  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion
```
What do you think?

Reply:
You can disable this linter rule, we cannot control all our dependencies so at some point if we reject `any`s and all `!` we'll be blocked, or we'll have to rely on `as` which is not a good practise. It is fine disabling this rule, but keeping the `any` one.

Reply:
b7460b1c75e76aefa5df57099a8cbecb7c46e999